ivre/passive.py

```diff
@@ -265,6 +265,16 @@ def _prepare_rec(spec, ignorenets, neverignore):
                 "md5",
                 clientvalue.encode(),
             ).hexdigest()
+    # Check DNS Blacklist answer
+    elif spec['recontype'] == 'DNS_ANSWER':
+        if any(spec['value'].endswith(dnsbl)
+               for dnsbl in config.DNS_BLACKLIST_DOMAINS):
+                dnsbl_val = spec['value']
```

Comment:
Maybe this could be factorized with `._update_dns_blacklist()` (not sure if this would help, we need to discuss that)?

Reply:
It is possible. 
If we use an option in `_update_dns_blacklist()` to determine if it is a creation or a modification of the current db.